### PR TITLE
add comments from DS-JK

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

Add comments from Doug and Jung, fix one typo

### Issue

NA

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

File: content/06.methods.md [hunk 1]
```diff
@@ -114,9 +114,9 @@ References can be obtained from the [Broad Genome References on AWS](https://s3.
 
 ##### Assessment of germline variant pathogenicity
 
-For patients with hypermutant samples, we first added population frequency of germline variants using `ANNOVAR` [@doi:10.1093/nar/gkq603] and pathogenicity scoring from ClinVar [@doi:10.1093/nar/gkt1113] using `SnpEff` [@doi:10.4161/fly.19695].
-We then filtered for variants with read depth >= 15, variant allele fraction >= 0.20, and which were observed at < 0.1% allele frequency across each population in the Genome Aggregation Database (see **Key Resources**).
-Finally, we retained variants in genes included in the KEGG MMR gene set (see **Key Resources**), _POLE_, and/or _TP53_ which were ClinVar-annotated as pathogenic (P) or likely pathogenic (L/P) and with review status of >= 2 gold stars.
+For patients with hypermutant samples, we first added population frequency of germline variants using `ANNOVAR` [@doi:10.1093/nar/gkq603] and pathogenicity scoring from ClinVar [@doi:10.1093/nar/gkt1113] using `SnpSift` [@doi:10.3389/fgene.2012.00035].
```
Comment (sjspielman) [inline content/06.methods.md:117]: So it looks like `SnpSift` called `SnpEff` under the hood? eg, 
https://github.com/d3b-center/OpenPBTA-workflows/blob/1855bc39d0a0b008d99c5d1d8b8b1f0093fb5659/cwl/clinvar_pathogenic_filter.cwl#L87-L92

Comment (sjspielman) [reply content/06.methods.md:117]: If this change goes through, we will need to update the KR table:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/93422de6526e87b7f6dc9e3617efd440411314b1/tables/output_tables.Rmd#L501

**Review by sjspielman — COMMENTED** [review]
Can we also add `Writing - Review and editing` to their contributions in metadata?

**Comment by jharenza** [comment]
> Can we also add `Writing - Review and editing` to their contributions in metadata?

I updated these for everyone in #330

File: content/06.methods.md (hunk 1 above)
Comment (jharenza) [reply content/06.methods.md:117]: Ahh good point- yes it was a comment from Jung that the part of the alg that does the annotation is SnpSift

Comment (sjspielman) [reply content/06.methods.md:117]: I opened https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1611 to track.

**Review by sjspielman — APPROVED** [review]

**Pull request merged**